docs/source/etcd-data-model.rst

```diff
@@ -40,8 +40,19 @@ endpoints
   (e.g. if it has multiple vNICs). See :ref:`endpoint-data` for more.
 
 security profiles
-  A security profile is an object that encapsulates a specific security policy
-  that can be applied to endpoints. See :ref:`security-profile-data` for more.
+  A security profile is an object that encapsulates a specific set of security
+  rules to apply to an endpoint.  Endpoints reference one or more security
+  profiles explicitly.  See :ref:`security-profile-data` for more.
+
+security policies
+  Similarly, security policies contain a set of security rules to apply.
+  However, security policies provide a richer, tiered security model, which
+  can override the security profiles directly referenced by an endpoint.
+
+  Security policies are divided into tiers, which are applied in order by
```

Comment:
A clearer alternative, I think: 'Each security policy has a selector predicate, such as "host is calico=vm13", that picks out the endpoints it should apply to, and an ordering number that specifies the policy's priority.  For each endpoint, Calico applies the security policies that apply to it, in priority order, and then that endpoint's security profiles.'

But it would be better if the example had two keys/values in it, and I'm sure you can think of a better example than mine.